packages/js/product-editor/src/components/attribute-control/attribute-skeleton.scss

```diff
@@ -0,0 +1,61 @@
+@mixin attributeSkeleton {
+	border-radius: $grid-unit-05;
+	min-width: $grid-unit-20;
+	min-height: $grid-unit-20;
+	color: transparent;
+}
+
+.woocommerce-product-page-attribute-skeleton {
+	border: 1px dashed $gray-200;
+	padding: 0 $gap-large 0 $gap-large;
+	&__row {
+		display: grid;
+		grid-template-columns: 45% auto 90px;
+		padding-bottom: $gap-large;
+		padding-top: $gap-large;
+		border-bottom: 1px solid $gray-200;
+	}
+	&__row:last-child {
+		border-bottom: none;
+	}
+	&__last-item {
+		justify-self: end;
+	}
+	&__row0 {
+		background-color: $gray-200;
+	}
+	&__row1 {
+		background-color: #e9e9e9;
+	}
+	&__row2 {
+		background-color: #efefef;
+	}
```

Comment:
You could handle this without separate classnames for each row, and instead use the `:nth-child()` CSS selector, but either approach works!

https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child

Reply:
oh! I will use this next time. It looks nicer. Thanks for the tip.